content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,773 @@
+---
+title: Ciclo de vida de un Pod
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+Esta página describe el ciclo de vida de un Pod.
+Los Pods siguen un ciclo de vida definido, comenzando en la fase [`Pending`](
+#pod-phase),
+y luego pasando a "en ejecución" `Running` si al menos uno de sus contenedores primarios se
+inicia correctamente,
+y luego pasando a "exitoso" (`Succeeded`) o "fallido" (`Failed`) si uno de los contenedores de un Pod
+termina en error.
+
+Mientras un Pod está en `Running`,
+el kubelet puede reiniciar sus contenedores para manejar algunos errores.
+Dentro de un Pod,
+Kubernetes rastrea diferentes [estados](#container-states) de contenedores y
```

Comment:
For "#" references inside this same file, we should put the [same words used below](https://github.com/kubernetes/website/pull/47204/files#diff-a69226e67855ca05d06aa3b8b30b84b4b7e837113a7ea4edc7ec685df4c06c66R115) by replacing the spaces by hyphens.

```suggestion
Kubernetes rastrea diferentes [estados](#estados-del-contenedor) de contenedores y
```

Reply:
Or, update the subtitle with the "#" reference:

```yaml
## Estados del contenedor {#container-states}
```

Reply:
I will update and maintain the link in the original language if you may.